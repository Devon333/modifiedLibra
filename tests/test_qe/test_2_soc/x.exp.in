&inputpp
  prefix = 'x',
  outdir = './',
  pseudo_dir = './',
  psfile(1) = 'C.rel-pbe-n-kjpaw_psl.0.1.UPF',
  psfile(2) = 'H.rel-pbe-kjpaw_psl.0.1.UPF',
  single_file = .FALSE.,
  ascii = .TRUE.,
  uspp_spsi = .FALSE.,
/
