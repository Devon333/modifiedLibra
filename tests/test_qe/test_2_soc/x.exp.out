
     Program PW_EXPORT v.5.2.0 starts on 14Jul2016 at 18: 0:16 

     This program is part of the open-source Quantum ESPRESSO suite
     for quantum simulation of materials; please cite
         "P. Giannozzi et al., J. Phys.:Condens. Matter 21 395502 (2009);
          URL http://www.quantum-espresso.org", 
     in publications or presentations arising from this work. More details at
     http://www.quantum-espresso.org/quote

     Serial version

     Reading data from directory:
     ./x.save

   Info: using nr1, nr2, nr3 values from input

   Info: using nr1, nr2, nr3 values from input

     IMPORTANT: XC functional enforced from input :
     Exchange-correlation      =  SLA  PW   PBX  PBC ( 1  4  3  4 0 0)
     Any further DFT definition will be discarded
     Please, verify this is what you really want

               file C.rel-pbe-n-kjpaw_psl.0.1.UPF: wavefunction(s)  2P 2P renormalized
               file H.rel-pbe-kjpaw_psl.0.1.UPF: wavefunction(s)  1S renormalized

     G-vector sticks info
     --------------------
     sticks:   dense  smooth     PW     G-vecs:    dense   smooth      PW
     Sum         225     225     61                 2553     2553     341


     Check: negative/imaginary core charge=   -0.000452    0.000000

     negative rho (up, down):  2.705E-01 0.000E+00

     PW_EXPORT    :     1.20s CPU         1.45s WALL


   This run was terminated on:  18: 0:17  14Jul2016            

=------------------------------------------------------------------------------=
   JOB DONE.
=------------------------------------------------------------------------------=
